src/main/java/umk/mat/pajda/ProjektZespolowy/controllers/ReviewDataController.java

```diff
@@ -24,7 +26,10 @@
     name = "Review Endpoints",
     description = "Controller for handling requests related to add/del/patch/read reviews")
 public class ReviewDataController {
+  @Value("${app.isProd:true}")
+  private boolean isProd;
 
+  private final String fixedSalt = "$2a$10$abcdefghijklmnopqrstuu";
```

Comment:
It should be stored in an env variable, but a better idea is to avoid having a single salt for all IPs. You can use BCrypt.gensalt() to get random salt.

Reply:
moved it to application.properties
BCrypt.gensalt() doesn't work for this use case

Reply:
i thought about something like adding variables, which are now in the readme.

Reply:
ok